pkg/services/ngalert/schedule/schedule.go

```diff
@@ -36,7 +36,7 @@ type ScheduleService interface {
 //
 //go:generate mockery --name AlertsSender --structname AlertsSenderMock --inpackage --filename alerts_sender_mock.go --with-expecter
 type AlertsSender interface {
-	Send(key ngmodels.AlertRuleKey, alerts definitions.PostableAlerts)
+	Send(ctx context.Context, key ngmodels.AlertRuleKey, alerts definitions.PostableAlerts)
```

Comment:
The only change here is adding a `context.Context` to the interface and the `.Send()` calls.
I also updated the mock for this interface.